docs/connecting.rst

```diff
@@ -0,0 +1,93 @@
+=========================
```

Comment:
Can we use a header format in *.rst files that does not depend on underlines/overlines? We had this problem before with AsciiDoc (which I fixed in https://github.com/JanusGraph/janusgraph/pull/115/commits/dc67ce73c08e79fa65b30bc8280861056070c573); it would be nice not to re-introduce the same issue with the docs again here.

Apparently, this is the Python style guide: https://devguide.python.org/documenting/#style-guide but if so, do we need to make sure that the length of overline and underline match the length of the header text itself?

Reply:
I guess this can't be helped. Also from link you shared it seems like underlines/overlines are only supported way in .rst files in Python.
As for matching the length, well, no the underlines and overlines doesn't need to match the length of text, but it **should not be short than title**. If the underlines/overlines **are longer than text** it still renders correctly. (See other sections like _Build JanusGraph Client_ or _Register Custom Serializer/Deserializer_)